iam.tf

```diff
@@ -68,7 +68,7 @@ resource "aws_iam_role_policy" "ext_tableau" {
 		"Action": [
 			"ssm:PutParameter"
 		],
-		"Resource": "arn:aws:ssm:eu-west-2:*:parameter/data_archive_tab_ext_backup_sub_directory"
+		"Resource": "arn:aws:ssm:eu-west-2:*:parameter/data_archive_tab_ext_staging_backup_sub_directory"
```

Comment:
What is the value of this parameter? Is it a local directory on the server or a keyprefix in S3?
If it is a keyprefix can it be named accordingly please?

Reply:
local directory on server.